app/routes/application.js

```diff
@@ -18,40 +18,15 @@ export default Route.extend(ApplicationRouteMixin, {
     this.get('authManager').initialize();
   },
   model() {
-    return RSVP.hash({
-      notifications: [
-        {
-          title       : 'New Session Proposal for event1 by  user1',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        },
-        {
-          title       : 'New Session Proposal for event2 by  user2',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : true
-        },
-        {
-          title       : 'New Session Proposal for event3 by  user3',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        },
-        {
-          title       : 'New Session Proposal for event4 by  user4',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : true
-        },
-        {
-          title       : 'New Session Proposal for event5 by  user5',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        }
-      ]
-    });
+    if (this.get('session.isAuthenticated')) {
+      return new RSVP.Promise((resolve, reject) => {
+        this.store.findRecord('user', this.get('authManager.currentUser.id'), { reload: true })
+          .then(user => {
+            user.get('notifications').then(resolve).catch(reject);
```

Comment:
@niranjan94 Sorry but, I didn't get that. We are showing read and unread notifications both in the dropdown and clicking notification will mark it as read. Why should we load the unread notifications then only? What I did here is loaded all the notifications(read and unread), passed two things to the template:

- all the notifications directly to show in the dropdown.
- filtered ones through the controller to show the number of unreads on the top of mail icon.

Please let me know if I am doing wrong and what is intended here.